src/sql/workbench/contrib/query/browser/query.contribution.ts

```diff
@@ -374,6 +374,11 @@ const queryEditorConfiguration: IConfigurationNode = {
 			'type': 'boolean',
 			'default': false,
 			'description': localize('queryEditor.promptToSaveGeneratedFiles', "Prompt to save generated SQL files")
+		},
+		'queryEditor.githubCopilotContextualizationEnabled': {
+			'type': 'boolean',
+			'default': false,
+			'description': localize('queryEditor.githubCopilotContextualizationEnabled', "Enable contextualization of SQL queries for GitHub Copilot. Helps GitHub Copilot to return improved suggestions, if installed.")
```

Comment:
Do you want to have it mention SQL in here if other providers (like postgres) are going to be looking into using this as well? Maybe also mention that this only applies to providers that have implemented contextualization...